.gitignore

```diff
@@ -0,0 +1,5 @@
+*.*~
```

Comment:
I think only this one is needed: iiuc, it tells git to ignore all files (in any dir) matching the pattern.

Reply:
looked it up and changed